docs/getting_started/setup/linux.md

```diff
@@ -3,7 +3,7 @@ Here's how you can install Crow on your favorite GNU/Linux distro.
 
 ### Requirements
  - C++ compiler with at least C++11 support.
- - boost library & development headers (1.64 or later).
+ - Asio development headers.
```

Comment:
Might be worth including the minimum ASIO version (`1.10.9` was the version included in boost `1.64`)

Reply:
I guess a minimum version should be included. What function limits the version? Why was boost 1.64 chosen?

Reply:
boost 1.64 was chosen I believe because that's the version conan used.

I looked at the changelog and it seems like 1.10.8 was the first to support OpenSSL 1.1, so 1.10.9 is either the oldest or second oldest version that Crow can work with.

[source](https://www.boost.org/doc/libs/1_79_0/doc/html/boost_asio/history.html#boost_asio.history.asio_1_10_9___boost_1_64)

Reply:
Then I'll make sure to mention it. 👍